common/unified-chart/templates/high-priority/analysis-template.yaml

```diff
@@ -0,0 +1,32 @@
+{{- $top := . -}}
+{{- $values := .Values -}}
+{{- if .Values.argoRollouts.progressiveDelivery.enabled }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: {{ include "hp.name" . }}
+spec:
+  args:
+    - name: service-name
+      value: {{ include "hp.name" . }}
+  metrics:
+    - name: succesful-black-box-test
+      successCondition: result == true
+      provider:
+        web:
+        {{- with .Values.argoRollouts.progressiveDelivery }}
+          method: POST # valid values are GET|POST|PUT, defaults to GET
```

Comment:
i suggest parameterizing this and moving to values.yaml under progressive delivery